examples/mt_pal_bert_tutorial_mrpc_rte.ipynb

```diff
@@ -0,0 +1,2933 @@
+{
+  "nbformat": 4,
```

Comment:
That's great. I've got a couple of questions.
1) Could you please add the reference to this ipynb to the documentation?
2) I am not sure that default logging of the model summary is nesessary. Maybe it's better either not to log it or make it by parameter?

Reply:
Added tutorial to docs
Made log model a parameter in `TorchModel`